applications/nrf5340_audio/src/modules/display/button_tab.c

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <display/display.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/gpio.h>
+#include <lvgl.h>
+#include <stdio.h>
+#include <string.h>
+#include <zephyr/kernel.h>
+#include <zephyr/zbus/zbus.h>
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+#include <stdint.h>
+#include "button_tab.h"
+#include <C:\ncs\v2.4.0\nrf\applications\nrf5340_audio\src\modules\hw_codec.h>
+#include <math.h>
+
+LOG_MODULE_REGISTER(nrf5340audiodk_display, CONFIG_NRF5340AUDIODK_DISPLAY_LOG_LEVEL);
+
+enum button_action {
+	BUTTON_PRESS,
+	BUTTON_ACTION_NUM,
+};
+struct button_msg {
+	uint32_t button_pin;
+	enum button_action button_action;
+};
+
+ZBUS_CHAN_DECLARE(button_chan);
+
+void button_create(lv_obj_t *current_screen, char label_text[20], lv_align_t *button_position,
+		   void (*button_callback_function)(lv_event_t *e))
```

Comment:
```suggestion
void button_create(lv_obj_t *current_screen, char *label_text, size_t lv_align_t,
                               void (*button_callback_function)(lv_event_t *e))
```
It is more common to pass strings as `char` pointers. I'm not completely sure it will work with the `LVGL` API though, so if it doesn't work we might not be able to do it.

Reply:
Also, this function isn't meant to be used outside of this file, so it can be declared static